cluster/v1/types.go

```diff
@@ -105,6 +109,15 @@ type ManagedClusterVersion struct {
 	Kubernetes string `json:"kubernetes,omitempty"`
 }
 
+// ManagedClusterClaim represents a ClusterClaim collected from a managed cluster.
+type ManagedClusterClaim struct {
+	// Name is a well known or customized name to identify the claim.
```

Comment:
give an example of the format of the name.  Also, if there is a regex for this and value in the upstream API, specify it here too.

Reply:
Examples and length restriction added.